draft-tjhai-ipsecme-hybrid-qske-ikev2-02.xml

```diff
@@ -383,10 +415,51 @@
 		needs IKE to handle any possible fragmentation, then he uses the protocol
 		listed below.</t>
 
+    <section title="IKE_SA_INIT Round: Negotiation" >
+        <t> Using multiple key exchanges is negotiated by means of standard IKEv2 mechanism - via SA payload.
+        For this purpose several new transform types, namely Additional Key Exchange 1, Additional Key Exchange 2,
+        Additional Key Exchange 3 etc. are defined. They are collectively called Additional Key Exchanges 
+        and have slightly different semantics than existing IKEv2 transform types.
+        They are interpreted as additional key exchanges that peers agreed to perform
+        in a series of INTERMEDIATE exchanges. The possible transform IDs for these transform types 
+        are the same as IDs for the transform type 4 (Diffie-Hellman Group), so they all share
+        a single IANA registry for transform IDs.
+        </t>
+        <t> Key exchange method negotiated via transform type 4 MUST always take place
+        in the IKE_SA_INIT exchange. Additional key exchange methods negotiated via newly
```

Comment:
```suggestion
        in the IKE_SA_INIT exchange. Additional Key Exchanges negotiated via newly
```
Since we already defined `Additional Key Exchanges`, could we use it here?